core/src/main/java/io/questdb/cairo/TableNameRegistryRW.java

```diff
@@ -162,7 +164,8 @@ private boolean renameToNew(TableToken oldToken, TableToken newToken) {
             nameStore.logAddTable(newToken);
 
             // Release the new name in the map.
-            assert tableNameToTableTokenMap.remove(oldToken.getTableName(), LOCKED_DROP_TOKEN);
+            boolean removed = tableNameToTableTokenMap.remove(oldToken.getTableName(), LOCKED_DROP_TOKEN);
```

Comment:
this code does the same thing as `public void unlockTableName(TableToken tableToken)` - perhaps it would be clearer and self explanatory to use his function?

Reply:
there is also a race condition here:
- if you put sufficient enough delay after L167
- create table by the "old" name - it will get renamed
- you can two tables renamed for the price of one